nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -132,49 +135,85 @@ def _get_input_edges_mapping(nncf_graph: NNCFGraph):
         return get_input_edges_mapping(nncf_graph)
 
     @staticmethod
-    def _get_axis(
-        nncf_graph: NNCFGraph, target_point: ONNXTargetPoint, quantizer_config: QuantizerConfig
+    def _get_quantized_channel_axis(
```

Comment:
What does this method do? Which axis would be used while conversion in case of a `None` result?

Reply:
Please take a look at the updated methods